lib/Helper/CsvFormulaFormatter.php

```diff
@@ -19,15 +19,8 @@
 
 class CsvFormulaFormatter extends \League\Csv\EscapeFormula
 {
-    public function unEscapeField(string $field): string
+    public function unEscapeField(mixed $field): string
     {
-        if (isset($field[0], $field[1])
-            && $field[0] === $this->getEscape()
-            && in_array($field[1], $this->getSpecialCharacters())
-        ) {
-            return ltrim($field, $field[0]);
-        }
-
-        return $field;
+        return parent::unescapeField($field);
     }
```

Comment:
Do we actually need this, if it's just calling the parent method?

Reply:
Reverted in bugfix. We should mark this deprecated in 11.1 and suggest to use the one from League\Csv

Reply:
Done in https://github.com/pimcore/pimcore/pull/16002/